proposals/extensions.md

```diff
@@ -500,7 +497,7 @@ We modify the [accessibility constraints](https://github.com/dotnet/csharpstanda
 
 The following accessibility constraints exist:
 - [...]
-- \***The underlying type of an extension type shall be at least as accessible as the extension type itself.**
+- ***The underlying type of an extension type shall be at least as accessible as the extension type itself.**
```

Comment:
Nit: it seems excessive to use three `*` - the first is rendered as a literal `*` and the remaining two make the text bold:

![image](https://github.com/dotnet/csharplang/assets/3669664/de7429fe-4677-49a3-8680-a04f76a472e5)

Reply:
This is intended. If found that the bold is not always easy to see, so the `*` helps spot additions even when the bold isn't.
It looks like GitHub may have adjusted their fonts and the bold seems easier to see than before, but I think it's better to keep for now. It'll be easy to remove if we decide this extra formatting isn't necessary.